### ISSUE TITLE
Not catching Invalid token or if a user is missing rights to do something

### ISSUE THREAD

**Issue by KivraChristoffer** [jagter/python-netbox#38]
So did by misstake use our read token and it took me a long time to find that.
That got me thinking and i did a test with a bad token and its not displayed in any way so its super hard to find out.

So in connection.py i just added a line to print 

`        print(response_data, response.status_code)
        return response.ok, response.status_code, response_data
`

```
'ipython3 1.py
'/home/chrnic/.local/lib/python3.8/site-packages/netbox/connection.py:39: SyntaxWarning: "is not" with a literal. Did you mean "!="?
  if method is not 'GET':
/usr/lib/python3/dist-packages/urllib3/connectionpool.py:999: InsecureRequestWarning: Unverified HTTPS request is being made to host 'netbox.FAKE.net'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
{'detail': 'Invalid token'} 403
```

I dont know where it would be proper to catch and handle bad token or "unauthorized"

**Comment by jagter** [comment]
I'm working on a solution. This will be fixed in the next release.

**Comment by jagter** [comment]
Fixed in 0.0.19
